package.json

```diff
@@ -6,7 +6,9 @@
     "dev": "turbo dev",
     "lint": "turbo lint",
     "test": "turbo test",
-    "format": "prettier --write \"**/*.{ts,tsx,md}\""
+    "format": "prettier '**/*.{ts,tsx,md,mdx,css}'",
+    "format:write": "pnpm format --write ",
```

Comment:
Changed from `pnpm run format --write` to `pnpm format --write` as the pipeline was not working otherwise. It seems to work locally with or without `run`.